site/docs/master/customize-installation.md

```diff
@@ -37,25 +37,9 @@ If you've already run `velero install` without the `--use-restic` flag, you can
 By default, the Velero deployment requests 500m CPU, 128Mi memory and sets a limit of 1000m CPU, 256Mi.
 Default requests and limits are not set for the restic pods as CPU/Memory usage can depend heavily on the size of volumes being backed up.
 
-If you need to customize these resource requests and limits, you can set the following flags in your `velero install` command:
-
-```bash
-velero install \
-    --provider <YOUR_PROVIDER> \
-    --bucket <YOUR_BUCKET> \
-    --secret-file <PATH_TO_FILE> \
-    --velero-pod-cpu-request <CPU_REQUEST> \
-    --velero-pod-mem-request <MEMORY_REQUEST> \
-    --velero-pod-cpu-limit <CPU_LIMIT> \
-    --velero-pod-mem-limit <MEMORY_LIMIT> \
-    [--use-restic] \
-    [--restic-pod-cpu-request <CPU_REQUEST>] \
-    [--restic-pod-mem-request <MEMORY_REQUEST>] \
-    [--restic-pod-cpu-limit <CPU_LIMIT>] \
-    [--restic-pod-mem-limit <MEMORY_LIMIT>]
-```
-
-Values for these flags follow the same format as [Kubernetes resource requirements][5].
+To customize these resource requests and limits, you can set the following flags in your `velero install` command:
```

Comment:
Looks like L40 should be removed.

Reply:
Thanks for catching that. I wanted to reword it in a new line and forgot to delete the stale one.